tests/test_networkx/test_nx_basics.py

```diff
@@ -155,19 +170,365 @@ def test_write_nx_metadata_override_precedence(tmp_path):
     with pytest.warns(UserWarning):
         geff.write_nx(
             graph,
-            path,
+            store=path,
             metadata=metadata,
             axis_names=["x", "y", "z"],  # Override with different axes
             axis_units=["meter", "meter", "meter"],
             axis_types=["space", "space", "space"],
         )
 
     # Verify that axis lists took precedence
-    read_graph, read_metadata = geff.read_nx(path)
+    _, read_metadata = geff.read_nx(path)
     assert len(read_metadata.axes) == 3
     axis_names = [axis.name for axis in read_metadata.axes]
     axis_units = [axis.unit for axis in read_metadata.axes]
     axis_types = [axis.type for axis in read_metadata.axes]
     assert axis_names == ["x", "y", "z"]
     assert axis_units == ["meter", "meter", "meter"]
     assert axis_types == ["space", "space", "space"]
+
+
+def test_create_simple_2d_geff():
```

Comment:
I think all the new tests from here down should be moved into a dedicated file for testing testing.utils.data